lib/management.js

```diff
@@ -3,12 +3,44 @@ const forge = require('node-forge');
 const arsenal = require('arsenal');
 const config = require('../conf/Config');
 const AWS = require('aws-sdk');
-
-const MetadataFileClient = arsenal.storage.metadata.MetadataFileClient;
-const mdClient = new MetadataFileClient({
-    host: config.queuePopulator.dmd.host,
-    port: config.queuePopulator.dmd.port,
-});
+const Logger = require('werelogs').Logger;
+
+const logger = new Logger('mongoclient');
+
+const serviceBucket = 'PENSIEVE';
+
+let setup = false;
```

Comment:
I think it could be clearer to define this in the function `initManagement` where it is used and redefined.